source/common/protobuf/yaml_utility.cc

```diff
@@ -325,22 +325,22 @@ ProtobufWkt::Value ValueUtil::loadFromYaml(const std::string& yaml) {
 
 namespace {
 
-char* UTF8CoerceToStructurallyValid(absl::string_view str, char* dst, const char replace_char) {
-  // Copy the whole string (unless src == dst).
-  if (str.data() != dst) {
-    memmove(dst, str.data(), str.size());
-  }
-  char* const initial_dst = dst;
+// This is a modified copy of the UTF8CoerceToStructurallyValid from the protobuf code.
+// A copy was needed after if was removed from the protobuf.
+// This function returns a boolean indicating that
```

Comment:
nit: I think this line should probably be removed?

Reply:
Done